lua/packer/luarocks.lua

```diff
@@ -166,17 +166,26 @@ end
 local luarocks_keys = {
   only_server = 'only-server',
   only_source = 'only-sources',
-  server = 'server'
 }
 
+local function is_valid_luarock_key(key)
+  return not (key == 'tree' or key == 'local')
+end
+
 local function format_luarocks_args(package)
   if type(package) ~= 'table' then return '' end
   local args = {}
   for key, value in pairs(package) do
-    local luarock_key = luarocks_keys[key]
+    if type(key) ~= "string" or not is_valid_luarock_key(key) then goto continue end
```

Comment:
Lua has labels and `goto`?? I've never seen this in the language reference.

As a small nitpick, I'd generally prefer if we inverted this conditional to avoid the `goto`, e.g.

```lua
if type(key) == 'string' and is_valid_luarock_key(key) then
....do the things....
end
```
but it's good to learn that `goto` is an option.

Reply:
@wbthomason I *think* based on a commit I saw from TJ in Telescope earlier that maybe `goto`s aren't specifically lua 5.1 syntax and maybe shouldn't be used if a plugin is assuming 5.1 so maybe a good idea to remove it anyway